docs/source/stepfunctions.rst

```diff
@@ -0,0 +1,118 @@
+.. _stepfunctions:
+
+CfnCluster Stepfunctions
+########################
+
+Why Stepfunctions
+=================
+
+* Allows for complex workflows with CfnCluster
+* Handles cluster creation, teardown, and updates
+* Useful for conditional automated job execution
+* Has better interfacing with other AWS services
+
+.. image:: https://s3.amazonaws.com/global-cfncluster/doc-images/parallel_job.gif
+
+Using CfnCluster Step Functions
```

Comment:
Getting Started

Reply:
are you telling my I could have saved 2 minutes while writing these docs??

Reply:
I got started changing this one

Reply:
need a getting started section to learn how to change these docs tbh